_data/patcher-landing-page-tabs-terraform-upgrade.yml

```diff
@@ -0,0 +1,7 @@
+- id: pr
+  title: PR
+  content: https://i.imgur.com/XDAl1Ew.png
+
+- id: patch
+  title: Patch
```

Comment:
Perhaps `tf-14-to-15.patch` would be a better title?

Reply:
Done in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/583/commits/ddca18c099444c77db94c7103c0a69cba8ea163f.